probe-rs/src/probe/cmsisdap/tools.rs

```diff
@@ -62,7 +62,7 @@ fn get_cmsisdap_info(device: &Device<rusb::Context>) -> Option<DebugProbeInfo> {
         .ok();
 
     // All CMSIS-DAP probes must have "CMSIS-DAP" in their product string.
-    if prod_str.contains("CMSIS-DAP") {
+    if prod_str.contains("CMSIS-DAP") || prod_str.contains("WCH-Link") {
```

Comment:
Anyway what are you suggesting to do there?
There is no `hidapi`, only `rusb`.

Reply:
@adamgreig , @Yatekii  ... are you able to provide some clarification to @burrbull?

Reply:
I am not sure I understand the question. You should still be able to check the interface string even if you cannot work with the device thereafter.